libbeat/_meta/fields.ecs.yml

```diff
@@ -224,6 +224,21 @@
           description: >
             Destination domain.
 
+        # Metrics
```

Comment:
I'm hopeful we can get https://github.com/elastic/ecs/pull/179 merged so that they are part of ECS.

I can move them over the Packetbeat's fields.common.yml. But is it problematic to use fields in the `source` and `destination` namespaces that are not in ECS?

Reply:
These fields are in ECS now. 🍾